app/code/Magento/GiftMessageGraphQl/README.md

```diff
@@ -0,0 +1,3 @@
+# GiftMessageGraphQl
+
+**GiftMessageGraphQl**
```

Comment:
Please, provide a short description of the module purpose

Reply:
**Done**